src/Actions/Categories/CreateCategories.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Rapidez\Statamic\Actions\Categories;
+
+use Illuminate\Database\Eloquent\Collection;
+use Illuminate\Support\Str;
+use Statamic\Facades\Entry;
+
+class CreateCategories
+{
+    public function create(Collection $categories, string $storeCode): \Illuminate\Support\Collection
+    {
+        if ($categories->isEmpty()) {
+            return new Collection();
+        }
+
+        return $categories->filter(fn ($category) => !blank($category->name) && !blank($category->url_path) && !blank($category->entity_id))->map(fn ($category) => [
+            'magento_entity_id' => $category->entity_id,
+            'magento_parent_id' => $category->parent_id ?? '',
+            'title' => $category->name,
+            'slug' => Str::replace('/', '-', $category->url_path),
+            'path' => $category->url_path ?? '',
+        ])->each(function ($category) use ($storeCode) {
```

Comment:
if you use a shorhand function, you can remove the use.

Reply:
Gotcha, gonna check the rest as well!